R/learner_stats_regr_glm.R

```diff
@@ -79,9 +82,13 @@ LearnerRegrGlm = R6Class("LearnerRegrGlm",
       if ("weights" %in% task$properties) {
         pars = mlr3misc::insert_named(pars, list(weights = task$weights$weight))
       }
+      # add family to parameters
       fam <- mlr3misc::invoke(get(pars$family), .args = self$param_set$get_values(tags = "family"))
       pars <- mlr3misc::insert_named(pars, list(family = fam))
-
+      # if na.action not specified, use system default
+      if (!"na.action" %in% pars$values) {
+        pars <- mlr3misc::insert_named(pars, list(na.action = getOption("na.action")))
+      }
```

Comment:
You can actually omit this as `mlr3misc::invoke` automatically fills in the defaults if missing from paramset, same as predict below.